### PR TITLE
Add support for bandwidth limitation in noisydgram BIO filter

### PR DESCRIPTION
**Pull request by t8m**
##### Checklist

- [x] tests are added or updated

### CODE REVIEW

**Review by hlandau — APPROVED** [review]
Looks good!

**Comment by t8m** [comment]
@hlandau Your approval seems cleared? Please re-approve if that was just by mistake.

@openssl/committers ping for second approval.

**Review by hlandau — APPROVED** [review]

**Comment by t8m** [comment]
Merged to the master branch. Thank you for the reviews.

**Pull request closed without merging**
